views/navigation.json

```diff
@@ -5,19 +5,86 @@
     "config": {
         "navigation": [
             {
-                "title": "Data Compliances",
+                "title": "Resources",
```

Comment:
Remove this navigation from HIPPA and GDPR

Reply:
This is done in both HIPAA and GDPR SPs